bundles/org.eclipse.mita.program/src/org/eclipse/mita/program/typesystem/ProgramLinker.xtend

```diff
@@ -12,6 +13,7 @@ class ProgramLinker extends PlatformLinker {
 		|| PlatformPackage.eINSTANCE.systemResourceEvent.isSuperTypeOf(classifier)
 		|| TypesPackage.eINSTANCE.event.isSuperTypeOf(classifier)
 		|| PlatformPackage.eINSTANCE.configurationItem.isSuperTypeOf(classifier)
+		|| EcorePackage.eINSTANCE.EObject.isSuperTypeOf(classifier)
```

Comment:
I need to add this, so that cross references in feature calls like `var x = mysensor.mymodality.read()` are also linked. This is the requirement for the CrossReferences to find referenced resource when building the constraint system.

I am wondering why we need these specific Linkers at all. What harm does it do when it links everything (no shouldLink() method).